dimagi/utils/couch/migration.py

```diff
@@ -64,20 +64,20 @@ def _migration_automatically_handle_dups(self):
     def _migration_get_sql_object(self):
         cls = self._migration_get_sql_model_class()
         try:
-            return cls.objects.get(couch_id=self._id)
+            return cls.objects.get(**{cls.couch_id_name: self._id})
```

Comment:
it looks like this class doesn't define a `couch_id_name` property so this won't work as is